### ISSUE TITLE
[Security Issue] JQuery 1.2 < 3.5.0 Multiple XSS

### ISSUE THREAD

**Issue by cysea** [PrivateBin/PrivateBin#864]
## Steps to reproduce

![image](https://user-images.githubusercontent.com/59068073/146562071-76a084c0-e365-4cfe-ae93-2bc5fab1eeb4.png)
Privatebin is currently using JQuery 3.4.1 which seems to be outdated, security wise. 

We should bump up the version. (Note that I have tried using `jquery-3.6.0.min.js`, under the name `jquery-3.4.1.js` on my instance, and it does break privatebin)

**PrivateBin version:** Latest
**Dockerfile:**
```
FROM php:apache

RUN curl -L https://github.com/PrivateBin/PrivateBin/archive/master.zip > /var/www/master.zip && \
...
```

**Comment by elrido** [comment]
This is a duplicate of #198 & #608 and blocked by #728 - we tried doing even a dot-upgrade and the bootstrap 3 template broke. If someone could update the bootstrap template to v5, jQuery could finally get updated and/or removed. Help wanted.
